spec/mailers/producer_mailer_spec.rb

```diff
@@ -103,6 +104,22 @@
     expect(mail.body.encoded).not_to include p5.name
   end
 
+  context "when a cancelled order has been resumed" do
+    let!(:order_resumed) do
+      order = create(:order, distributor: d1, order_cycle: order_cycle, state: 'complete')
+      order.line_items << create(:line_item, variant: p6.variants.first)
+      order.finalize!
+      order.cancel
+      order.resume
+      order.save
```

Comment:
I like to use `save!` in specs to make sure that we detect failures here.

Reply:
Updated :ok_hand: